tstest/integration/integration_test.go

```diff
@@ -543,7 +543,17 @@ func (n *testNode) AwaitResponding(t testing.TB) {
 	t.Helper()
 	n.AwaitListening(t)
 
-	st := n.MustStatus(t)
+	var st *ipnstate.Status
```

Comment:
This seems unrelated. Revert? Or at least put in a separate commit?

Reply:
Oops, this was a change I thought I needed during earlier debugging, but turned out that no.